libportability-gfx/src/handle.rs

```diff
@@ -51,3 +53,53 @@ impl<T> fmt::Debug for Handle<T> {
         write!(formatter, "Handle({:p})", self.0)
     }
 }
+
+#[repr(C)]
+pub struct DispatchHandle<T>(u32, Handle<T>);
```

Comment:
would be nice to comment cleanly on the difference between `Handle` and `DispatchHandle`